storybookjs/storybook
# Fix ng component prop output override

Issue:
App component overrides output component props with handler function instead of subscribe on event.
Output properties should be EventEmitter instance, and function that defined in story, should be subscribed to that event.

## What I did
app.component now subscribe handler fn on output EventEmitter if prop is function and output is EventEmitter. Fixed button.component in demo

## How to test
Run angular example. Check output property type in component. It should be EventEmitter instance.

Is this testable with jest or storyshots?
no

Does this need a new example in the kitchen sink apps?
no

Does this need an update to the documentation?
no

If your answer is yes to any of these, please make sure to include it in your PR.
